app/controllers/bookmarks_controller.rb

```diff
@@ -132,23 +132,22 @@ def status_action documents
     status = params['action']
     Array(documents.map(&:id)).each do |id|
       media_object = MediaObject.find(id)
-      if cannot? :update, media_object
-        errors += ["#{media_object.title} (#{id}) #{t('blacklight.messages.permission_denied')}."]
+      if status == 'publish'
+        success_ids << id if can?(:update, media_object)
```

Comment:
ah its in the loop...so bascially for that item it just gets skipped over with no messaging etc. Is that desired?